### DISCUSSION TITLE
The last update broke something.

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#1968] [General]
The last update broke something.

![Screenshot](https://user-images.githubusercontent.com/99086477/215533671-021bd6fb-a4ae-4882-8ac8-a11160427adb.png)

**Comment by MikleBas** [comment]
UPD
The script asked me to enter this command, git config --global --add safe.directory /www/wwwroot/MY_SITE_DIRECTORY
git pull worked
  ./update_cloudlog.sh too
But since I automatically receive new data from this site with your software from this site, something in the background has broken

**Reply by phl0** [reply]
Ah ok. That was due to the git debug info on that screen. Let me check how/if we can disable the asking for a config entry.

**Comment by MikleBas** [comment]
I am ready to give commands or their output on the server
listen to you

**Comment by magicbug** [comment]
Not sure what you thinks broken? the Git info will only show if PHP can execute commands and that's permissions based.

**Comment by phl0** [comment]
The thing is: If the .git directory does not belong to the user the web server runs under it throws the error message about the unsafe directory and the git info is not shown on the debug page within Cloudlog. I changed the group permissions for .git to my user account that I do the git push/pull work with and it works. Debug info is shown without error messages.

**Reply by MikleBas** [reply]
But I did not change anything either before or after the first installation of the scripts
True, on the day when this message appeared on the server, the version of Git was updated - perhaps because of this?
git --version
git version 2.30.2

![image](https://user-images.githubusercontent.com/99086477/215774426-529d5ca3-e6d8-40e7-bfe7-61b347eac479.png)

**Reply by phl0** [reply]
Jepp. This was added to git (don't know which version). So its possible that it worked before without error messages.
